src/Preloader.php

```diff
@@ -146,17 +146,17 @@ protected function canGenerate(string $path, bool $overwrite) : bool
      * Writes the preloader script file into the specified path.
      *
      * @param  string  $path
-     * @return false|int
+     * @return true
      *
      * @codeCoverageIgnore
      */
     protected function performWrite(string $path)
     {
-        if (file_put_contents($path, $this->prepareCompiler($path)->compile(), LOCK_EX)) {
-            return true;
+        if (file_put_contents($path, $this->prepareCompiler($path)->compile(), LOCK_EX) === false) {
+            throw new RuntimeException("Preloader couldn't write the script to [$path].");
         }
```

Comment:
Put the unhappy path in the condition.

[_source_](https://github.com/szepeviktor/very-clean-code#-happy-and-unhappy-path)